sdk/monitor/azquery/autorest.md

```diff
@@ -13,6 +13,7 @@ openapi-type: "data-plane"
 output-folder: ../azquery
 security: "AADToken"
 use: "@autorest/go@4.0.0-preview.60"
+inject-spans: true
```

Comment:
Is this a new autorest command for the newest code generator version?

Reply:
Noticed we didn't add this when adding spans to keyvault.

Reply:
Yeah, it will inject span creation in the generated code. It's on by default for ARM, and we should also enable it for data-plane packages that directly expose the generated code.

Same setting is also enabled for the key vault packages (for `azkeys` it was enabled in the previous beta PR).

Reply:
Thanks! Makes sense, see that now.